include/vrapp/VRVolumeApp.h

```diff
@@ -1,28 +1,49 @@
 #ifndef VRVOLUMEAPP_H
 #define VRVOLUMEAPP_H
 
+
 #include <string>
 #include <vector>
 #include <future>
 #include <api/MinVR.h>
-#include "interaction/Labels.h"
-#include "interaction/ArcBall.h"
+
+#include "interaction/ArcBallCamera.h"
 #include "render/Volume.h"
 
-#include "Model.h"
+
+#include "ShaderProgram.h"
 
 
 class UIView;
 class CreateMovieAction;
 class VolumeRenderer;
 class DepthTexture;
 class Window_Properties;
+class Model;
+class Texture;
+class Simulation;
+class Labels;
+
+enum MovieState
```

Comment:
Are you going for ALL CAPS or CamelCase for enums? Stick to one

Reply:
renamed it using ALL CAPS